### ISSUE TITLE
Version_Control page is incomplete - it does not say where it is going to initialize a VCS folder or which of Git/Hg/Bazaar to choose from

### ISSUE THREAD

**Issue by paul-hammant** [jaap-karssenberg/zim-desktop-wiki#210]
Re: http://zim-wiki.org/manual/Plugins/Version_Control.html

I was expecting a checkbox in the editor panel so I could choose Git instead of Hg or Bazaar.

I was also expecting to be informed where the .git folder was made. I checked afterward and it was not in the Notebook's own folder (yes I did `ls -al`).

I'm confused....

**Comment by sojusnik** [comment]
Agreed!

To mention something like:

```
git config --global user.name USERNAME
git config --global user.email MAIL
```

would be very useful.

**Comment by Self-Perfection** [comment]
@sojusnik I disagree. git provides this help messaage already in case something went wrong. Zim should not include documentation on how to use any VCS, that would be reimplementing something that is already done somewhere.

But zim should hand over error message from git to the user. Does it? If zim does not display error message from git, that is the error that needs fixing. Not a lack of documentation help from zim.

**Comment by jaap-karssenberg** [comment]
I think indeed most robust solution is zim showing the git error output in an error message. 

Manual could add a note for first time setup, but rather link to chapter in git documentation than duplicate instructions.

**Comment by paul-hammant** [comment]
OP approves of this title change :)
